src/tools/calculate/definition.json

```diff
@@ -2,8 +2,7 @@
     "name": "calculate",
     "description": "Perform mathematical calculations including basic arithmetic, trigonometry, and advanced functions",
     "auto_run": true,
-    "requires_backup": false,
```

Comment:
if those are removed, how system knows if given command requires backup or not ? where in documentation this is described?